tests/MyGame/Example/Stat.java

```diff
@@ -41,5 +41,14 @@ public static int endStat(FlatBufferBuilder builder) {
     int o = builder.endObject();
     return o;
   }
+
+  public static final class Vector extends TableVector {
+    public void __init(int _vector_start_pos, int _vector_element_size, ByteBuffer _bb) { vector_start_pos = _vector_start_pos; vector_data_pos = __vector_data(_vector_start_pos); vector_element_size = _vector_element_size; bb = _bb; }
+    public Stat.Vector __assign(int _vector_start_pos, int _vector_element_size, ByteBuffer _bb) { __init(_vector_start_pos, _vector_element_size, _bb); return this; }
+
+    public Stat get(int j) { return get(new Stat(), j); }
+    public Stat get(Stat obj, int j) {  return obj.__assign(__indirect(vector_data_pos + j * vector_element_size, bb), bb); }
+    public int length() { return bb.getInt(vector_start_pos);}
```

Comment:
while we're caching, could cache the length as well?

Reply:
Fixed